security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -165,7 +196,7 @@ The following advisory is available for the OpenShift Compliance Operator 0.1.61
 
 * Before this update, the instructions for `ocp4-kubelet-configure-tls-cipher-suites` were incomplete, requiring users to refine the query manually. With this update, the query provided in `ocp4-kubelet-configure-tls-cipher-suites` returns the actual results to perform the audit steps. (link:https://issues.redhat.com/browse/OCPBUGS-3017[*OCPBUGS-3017*])
 
-* Before this update,`ScanSettingBinding` objects created without a `settingRef` variable did not use an appropriate default value. With this update, the `ScanSettingBinding` objects without a `settingRef` variable use the `default` value. (link:https://issues.redhat.com/browse/OCPBUGS-3420[*OCPBUGS-3420*])
+* Before this update,`ScanSettingBinding` objects created without a `settingRef` variable did not use an appropriate default value. With this update, the `ScanSettingBinding` objects without a `settingRef` variable use the `default` value.
```

Comment:
I wonder if there is a reason why those link ref are being removed

Reply:
Hi Vincent - the links are removed because they are internal bugs. This is part of our new tooling update to catch references that customers cannot access or are not publicly available. 

@GroceryBoyJr  we need to remove the entire entry, including the description of the bug.

Reply:
This is done....thank you for the feedback!